src/components/day-list.js

```diff
@@ -58,16 +58,19 @@ const EventCard = ({
     </div>
 }
 
+const short_months = ["", "ม.ค.", "ก.พ.", "มี.ค.", "เม.ย.", "พ.ค.", "มิ.ย.", "ก.ค.", "ส.ค.", "ก.ย.", "ต.ค.", "พ.ย.", "ธ.ค."];
+const thai_date = (str) => {
+  const dmy = str.split("-"); // NOTE the date string must be "dd-mm-yyyy"
+  return `${+dmy[0]} ${short_months[+dmy[1]]} ${+dmy[2] + 543}`;
+}
+
```

Comment:
Thanks for the PR. We could go with this approach for the beta version. 

Another approach that we might look at is to use `monent`'s i18n.

Reference: https://momentjs.com/docs/#/i18n/